sopel/plugins/callables.py

```diff
@@ -0,0 +1,586 @@
+"""Plugin objects' definition.
+
+.. versionadded:: 8.1
+
+The core of a plugin consists of several objects, either plugin callables or
+plugin jobs. These objects are represented by instances of
+:class:`PluginCallable` and :class:`PluginJob` respectively.
+
+.. seealso::
+
+    The :mod:`sopel.plugin`'s decorator create these objects directly.
```

Comment:
Englishing just a little :)

```suggestion
    The decorators in :mod:`sopel.plugin` create these objects directly.
```